### ISSUE TITLE
Panda::Date-related test failures (0.99_008)

### ISSUE THREAD

**Issue by eserte** [garu/Data-Printer#127]
On some of my smokers I see the following failure:
```
Locale en_US does not seem to be implemented on this system, keeping locale 
Locale en_US does not seem to be implemented on this system, keeping locale 

#   Failed test 'Panda::Date'
#   at t/100-filter_datetime.t line 288.
#          got: '2003-03-11 00:00:00 [EST]'
#     expected: '2003-02-11 00:00:00 [EST]'

#   Failed test 'Panda::Date (no timezone)'
#   at t/100-filter_datetime.t line 295.
#          got: '2003-03-11 00:00:00'
#     expected: '2003-02-11 00:00:00'

#   Failed test 'Panda::Date::Int'
#   at t/100-filter_datetime.t line 306.
#          got: '2003-03-11 00:00:00 [EST] ~ 2003-04-13 07:00:00 [EDT]'
#     expected: '2003-02-11 00:00:00 [EST] ~ 2003-03-13 07:00:00 [EST]'
# Looks like you failed 3 tests of 23.
t/100-filter_datetime.t .. 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/23 subtests 
# testing color level: 1
```

**Comment by garu** [comment]
Thanks @eserte! That's really helpful! I tried patching it with 6bb7c92 but Panda::Date is not installing here so I can't test this filter.

I've just made a new 0.99_009 release on CPAN with this patch, hopefully the issue is fixed. Could you let me know if you're still bumping into it on the new version?

Thanks again!

**Comment by eserte** [comment]
Yes, same problem with 0.99_009

**Comment by garu** [comment]
Missed a typo on the test, I think it's fixed for real this time on 0.99_010!

Please let me know if you still see any failures regarding Panda::Date.

Thanks!
